units/CMakeLists.txt

```diff
@@ -46,12 +69,14 @@ elseif(UNITS_BUILD_OBJECT_LIBRARY)
 
     if(UNITS_NAMESPACE)
         target_compile_definitions(units PUBLIC -DUNITS_NAMESPACE=${UNITS_NAMESPACE})
-
-        if(UNITS_BASE_TYPE)
-            target_compile_definitions(
-                units PUBLIC -DUNITS_BASE_TYPE=${UNITS_BASE_TYPE}
-            )
-        endif()
```

Comment:
Unreleated, but here is a useful tool, https://github.com/cheshirekow/cmake_format, basically clang-format but for cmake.

Reply:
I think that gets run as part of the automatic formatting sequence so I have used it.